src/Plugin/WebformElement/CivicrmContact.php

```diff
@@ -130,6 +130,9 @@ public function prepare(array &$element, WebformSubmissionInterface $webform_sub
       $element['#options'] = [];
       $element['#attributes']['data-is-select'] = 1;
     }
+    elseif ($element['#widget'] === 'autocomplete') {
+      $element['#attributes']['data-is-autocomplete'] = 1;
+    }
```

Comment:
Also, I think it would be nice to check for textfield instead of autocomplete since we do have some description and size attribute to attach to the contact id field. Eg:

```
diff --git a/src/Plugin/WebformElement/CivicrmContact.php b/src/Plugin/WebformElement/CivicrmContact.php
index 0dbf37a..e3d8a24 100644
--- a/src/Plugin/WebformElement/CivicrmContact.php
+++ b/src/Plugin/WebformElement/CivicrmContact.php
@@ -130,6 +130,11 @@ class CivicrmContact extends WebformElementBase {
       $element['#options'] = [];
       $element['#attributes']['data-is-select'] = 1;
     }
+    elseif ($element['#widget'] === 'textfield') {
+      $element['#attributes']['data-is-contactid'] = 1;
+      $element['#description'] = 'Lookup by Contact ID.';
+      $element['#size'] = 8;
+    }
     // get translated properties used for autocomplete widget
     $element['#attributes']['data-search-prompt'] = $this->getTranslatedElementProperty($element, 'search_prompt') ?? t('- Choose existing -');
     $element['#attributes']['data-none-prompt'] = $this->getTranslatedElementProperty($element, 'none_prompt') ?? t('+ Create new +');
```

Reply:
Added with 42e1fcc3ef594648252eb75e986deff6f5a1b821.